adaptive_components/example/lib/adaptive_column_example.dart

```diff
@@ -0,0 +1,134 @@
+// Copyright 2020, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:adaptive_components/adaptive_components.dart';
+
+void main() {
+  runApp(AdaptiveColumnsExample());
+}
+
+class AdaptiveColumnsExample extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        body: AdaptiveColumn(
+          columns: AdaptiveConstraintsColumn.adaptive(10),
+          children: [
+            AdaptiveContainer(
+              adaptiveConstraints: AdaptiveConstraints.xs(),
+              adaptiveColumn: 2,
+              color: Colors.red,
+              child: Text('This is an extra small window'),
+            ),
+            AdaptiveContainer(
+              adaptiveConstraints: AdaptiveConstraints.s(),
+              adaptiveColumn: 2,
+              color: Colors.orange,
+              child: Text('This is a small window'),
+            ),
+            AdaptiveContainer(
+              adaptiveConstraints: AdaptiveConstraints.m(),
```

Comment:
I think this won't actually compile since you renamed these:
```suggestion
              adaptiveConstraints: AdaptiveConstraints.medium(),
```

Reply:
Done